src/ConnectedMode.UnitTests/Migration/MigrationPromptTests.cs

```diff
@@ -55,7 +56,7 @@ public void CheckIsNonSharedMefComponent()
         public void Ctor_VerifySubscribed()
         {
             var migrationWizardController = new Mock<IMigrationWizardController>();
-            _ = new MigrationPrompt(Mock.Of<SVsServiceProvider>(), Mock.Of<INotificationService>() , migrationWizardController.Object, new NoOpThreadHandler());
+            _ = CreateTestSubject(migrationWizardController: migrationWizardController.Object);
```

Comment:
I changed the tests to use CreateTestSubject becuase it started to get cluttered. Dont know why i didnt do it in the first place but better late than never